.appveyor.yml

```diff
@@ -1,8 +1,15 @@
 environment:
   matrix:
-  - TARGET: x86_64-pc-windows-msvc
-  - TARGET: i686-pc-windows-msvc
-  - TARGET: i686-pc-windows-gnu
+    - TARGET: x86_64-pc-windows-msvc
+    - TARGET: i686-pc-windows-msvc
+    - TARGET: i686-pc-windows-gnu
+
+matrix:
+  allow_failures:
```

Comment:
I think so. I'll spelunk through their UI and see if I can turn this into a post merge check on master.